CHANGELOG.md

```diff
@@ -21,7 +21,10 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/workspace/compare/release-2.0.1...master
+[Unreleased]: https://github.com/atlassian/workspace/compare/release-2.0.2...master
+
+## [2.0.2] - 2019-11-04
+[2.0.2]: https://github.com/atlassian/workspace/compare/release-2.0.1...release-2.0.2
 
 ### Fixed
 - Support working in a Git Worktree folder. Fix [JPERF-667].
```

Comment:
As a drive-by-change, could you fix this broken link?

Reply:
- [ ] fix this broken link